setup.py

```diff
@@ -94,7 +94,7 @@ def find_version(*file_paths):
         'sqlparse==0.2.4',
         'pymongo>=3.2.0',
         'django>=2.0',
-        'dataclasses>=0.1',
+        'dataclasses==0.6',
```

Comment:
It should probably be:
`        'dataclasses>=0.6',`
for compatibility for future versions

Reply:
dataclasses>=0.7 only supports python3.6

Reply:
oh. didn't notice